index.html

```diff
@@ -147,40 +169,58 @@ <h2>Uploading New Materials</h2>
 			</div>
 			<br><br>
-		
+
+			<a name="Children"><div class="section">
+				<h2>Adding Children Learning Objects</h2>
+				<hr>
+				<p>When you upload a large learning object that uses many smaller objects in it, it is useful to add these objects as <i>children</i> to the larger one.  This is to help organize your larger learning objects and allow downloaders to easily find all the parts of your curriculum.</p>
+				<p>To start adding child objects go to the learning object builder, click on the <i>Add/Delete</i> button for Children, and click the option that says <i>Add Child +</i> . From here you have the option to search and choose which learning objects are the children of that parent object.  Note that the option to add children is available for all lengths of objects excluding nanomodules.  It is important to only add the direct children of the specific learning object you are working on. For example, the <i>CLARK Tutorial Learning Object</i> is a micromodule that has two children nanomodules. The <i>CLARK Tutorial Learning Object</i> micromodule could then be the child of a learning object that is either a module or a longer length object.</p>
+				<div class="picture">
+					<img src="./Resources/CLARK-Children-Objects.png" width="1000px">
+					<p style="padding-top: 20px;">Keep in mind adding children only builds the hierarchy for your learning objects.  You can edit an object&#39;s children at anytime by simply navigating to your dashboard and selecting the learning object you would like to make a revision on from the <i>Released</i> tab.</p>
+				</div>
+			</div>
+			<br><br>
+
 			<a name="Editing"><div class="section">
 				<h2>Editing Learning Objects</h2>
 				<hr>
-				<p>Go back to your learning objects dashboard. You will notice that the learning object you have just created has appeared in this list.</p>
-				<p style="padding-bottom: 20px;">To edit a learning object, click the 'meatball menu' (...) of the object you want to edit. There will be a list of different options, including an option to edit the learning object itself and manage the materials in it. Note: if your object is currently waiting or under review, you cannot access the meatball menu and, therefore, cannot edit your object.</p>
+				<p>Navigate to your dashboard. By default, you should see your draft learning objects under the <i>Draft</i> tab. You will find all learning objects that you have created that have not yet been released in this list.</p>
+				<p style="padding-bottom: 20px;">To edit a learning object, click the 'meatball menu' (...) of the object you want to edit. There will be a list of different options, including an option to edit the learning object itself and manage the materials in it. From here, click the <i>Edit</i> option and you will be redirected to the learning object creation page. You can edit or change as much as you desire. The learning object builder automatically saves your work, similar to a Google Document.</p>
```

Comment:
Instead of "You can edit or change as much as you desire. The learning object builder automatically saves your work, similar to a Google Document." I would say "While making chances in the learning object builder, your work will saved automatically, similarly to Google Documents."